#! /bin/sh
patch -p0 -f $* < $0
exit $?

To avoid problems with other UI servers (e.g.Kleopatra), we do not
want to start in server mode by default.


--- src/gpa.c   (revision 874)
+++ src/gpa.c   (working copy)
@@ -410,11 +410,14 @@
                                         keyservers->data);
     }
 
-  /* Fire up the server.  */
-  gpa_start_server ();
-
-  if (! args.start_only_server)
+  if (args.start_only_server)
     {
+      /* Fire up the server.  Note that the server allows to start the
+         other parts too.  */
+      gpa_start_server ();
+    }
+  else
+    {
       /* Don't open the keyring editor if any files are given on the
          command line.  Dito for the clipboard.   */
       if (args.start_keyring_editor && (optind >= argc))


